components/chat/ChatFeed.vue

```diff
@@ -33,6 +33,12 @@
               buttonText="IMG"
               cls="btn btn-outline-primary me-2 mt-2"
             />
+
+            <!-- Emoji Picker -->
+            <EmojiPicker  
+              v-if="isActivated && $config.web3storageKey !== '' && userStore.getIsConnectedToOrbis && isSupportedChain && hasDomainOrNotRequired"
```

Comment:
`$config.web3storageKey !== ''` is not needed here (`web3storage` is for uploading files)